src/jsonToGraphQLQuery.ts

```diff
@@ -17,8 +18,10 @@ export function jsonToGraphQLQuery({
 }) {
   const variablesData = {} as any
 
+  const newJsonQuery = cloneDeep(jsonQuery)
```

Comment:
here is how I fixed the issue.

I'm using `lodash` as it was already a project dependency.